build/uidp_helm/values.yaml

```diff
@@ -0,0 +1,98 @@
+# Default values for UIDProvider.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 6
+
+image:
+  repository: nginx
```

Comment:
@franciscobonand Let's set a specific SHA256 or version

Reply:
This is actually removable, not needed

Reply:
Removed it